src/dlgTriggerEditor.cpp

```diff
@@ -7078,6 +7084,261 @@ void dlgTriggerEditor::slot_debug_mode()
     mudlet::mpDebugArea->setWindowTitle("Central Debug Console");
 }
 
+void dlgTriggerEditor::slot_next_section()
+{
+    switch (mCurrentView) {
+    case EditorViewType::cmTriggerView:
+        if (QString("edbee::TextEditorComponent").compare(QApplication::focusWidget()->metaObject()->className()) == 0) {
```

Comment:
This should be a `QStringLiteral` as with others:

![Selection_427](https://user-images.githubusercontent.com/110988/63143173-4fe3e300-bfed-11e9-86e2-4107d186ac6f.png)

Reply:
Fixing in commit.